aspnetcore/host-and-deploy/windows-service.md

```diff
@@ -14,11 +14,11 @@ By [Luke Latham](https://github.com/guardrex) and [Tom Dykstra](https://github.c
 
 An ASP.NET Core app can be hosted on Windows as a [Windows Service](/dotnet/framework/windows-services/introduction-to-windows-service-applications) without using IIS. When hosted as a Windows Service, the app automatically starts after reboots.
 
-[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/host-and-deploy/windows-service/samples) ([how to download](xref:index#how-to-download-a-sample))
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/host-and-deploy/windows-service/) ([how to download](xref:index#how-to-download-a-sample))
 
 ## Prerequisites
 
-* [PowerShell 6](https://github.com/PowerShell/PowerShell)
+* [PowerShell 6.2 or later](https://github.com/PowerShell/PowerShell)
```

Comment:
Using New-Local* will require additional steps on older Operating systems (Older than RS4.)  You will have to manually import the module as it is not marked as compatible in the OS.

Reply:
I see ... it won't be documented when the `New-LocalUser` command is documented in the PowerShell docs? If it will be documented there, then I already link to the cmdlet doc.

Reply:
@rjmholt should know more about the documentation plan for the cmdlet compatibility in general

Reply:
Thanks much for your help @TravisEz13 ... *I'll 🛌 better tonight!*

Reply:
`New-LocalUser` is external to PowerShell 6.2; [Microsoft.PowerShell.LocalAccounts](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.localaccounts/new-localuser?view=powershell-5.1) is the only documentation for it. The module was written for Windows PowerShell, but as of Windows 1809 has been verified and marked as compatible with PowerShell 6.1+. But to be clear, Microsoft.PowerShell.LocalAccounts ships in Windows, not as part of PowerShell 6.

PowerShell 6.1+ understands this concept of compatibility as the `CompatiblePSEditions` of the module, which is explained [here](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_powershell_editions?view=powershell-6).

The TL;DR is that there are three scenarios:

- PowerShell 6.1 and above, Windows 1809 and above:
    - Microsoft.PowerShell.LocalAccounts has been verified compatible and will work as is
- PowerShell 6.1 and above, older Windows versions:
    - Module has not been assessed for compatibility and will not load without `Import-Module -SkipEditionCheck`. It's not recommended you do this, since older versions of the module may have problems. Just successfully loading the module doesn't mean commands will work.
    - Instead you should use the [WindowsCompatibility](https://github.com/PowerShell/WindowsCompatibility) module. Be aware that objects you get back from commands imported with this are remote objects and may not function the same way.
- PowerShell 6.0:
    - This version of PowerShell is out of support. The .NET Core version it runs on is incompatible with most of the Windows PowerShell modules, but they're not on the module path anyway.

Reply:
It seems that since I can call for 6.2+ as a prereq that I don't need the note at all. Is that correct?

Reply:
No note needed for PowerShell versions, but older Windows versions have a Microsoft.PowerShell.LocalAccounts version that's not compatible with PowerShell Core.

Instead you need to use the WindowsCompatibility module (or just use Windows PowerShell)

Reply:
Example of latter in PS 6:

```powershell
$name = '{NAME}'
powershell.exe -Command "New-LocalUser -Name $Name"
```

Reply:
I see ..... so I can add to PS 6.2+ that one needs Windows 10 April 2018 Update (version 1803/build 10.0.17134) or later. That should do the trick.